doc/api/https.md

```diff
@@ -28,7 +28,11 @@ An [`Agent`][] object for HTTPS similar to [`http.Agent`][]. See
 * `options` {Object} Set of configurable options to set on the agent.
   Can have the same fields as for [`http.Agent(options)`][], and
   * `maxCachedSessions` {number} maximum number of TLS cached sessions.
-    Use `0` to disable TLS session caching. **Default:** `100`.
+    Use `0` to disable TLS session caching. **Default:** `100`
```

Comment:
We usually do not use periods only if `**Default:**` part does not follow a full sentence.
```suggestion
    Use `0` to disable TLS session caching. **Default:** `100`.
```